identity-and-access-management/key-manager/api-cli/encrypt-decrypt-data-with-km-dek.mdx

```diff
@@ -0,0 +1,276 @@
+---
+meta:
+  title: Encrypting and decrypting data with a Key Manager data encryption key
+  description: Learn how to use Tink with Scaleway's Key Manager to securely encrypt and manage keys, including KEKs and DEKs, in Go applications.
```

Comment:
Here we talk only about Go, but the page explains how to use both Go and Python providers